### PR TITLE
Fix grammar for non/interactive session event log

### PR DESCRIPTION
**Pull request by kimlisa**
resolves https://github.com/gravitational/teleport/issues/4051
fixes https://github.com/gravitational/teleport/issues/4035

#### Description
- Make vertical/horizontal scrollbar appear on content overflow (small window size)
- Fix grammar for non/interactive session event log

#### Screenshot
grammar correction:
Screenshot from 2020-07-15 12-00-06
Screenshot from 2020-07-15 12-03-20
Screenshot from 2020-07-15 12-20-01

vertical/horizontal scrollbar:
screenshot_5

### CODE REVIEW

File: packages/teleport/src/services/audit/makeEvent.ts [hunk 1]
```diff
@@ -397,13 +397,13 @@ export const formatters: Formatters = {
         event.server_hostname || event.server_addr || event.server_id;
 
       if (!event.interactive) {
-        return `User [${user}] has ended non-interactive session [${event.sid}] on node [${node}] `;
+        return `User [${user}] has ended an non-interactive session [${event.sid}] on node [${node}] `;
```
Comment (webvictim) [inline packages/teleport/src/services/audit/makeEvent.ts:400]: This should be `a non-interactive session` rather than `an non-interactive session`

Comment (kimlisa) [reply packages/teleport/src/services/audit/makeEvent.ts]: revised: https://github.com/gravitational/webapps/pull/124/commits/dc32916bf82b41ba5dded6dd83f0510a42cec023

File: packages/teleport/src/cluster/components/Audit/AuditEvents/__snapshots__/AuditEvents.story.test.tsx.snap [hunk 2]
```diff
@@ -1396,7 +1528,7 @@ exports[`rendering of Events 1`] = `
           <td
             style="word-break: break-word;"
           >
-            User [admin@example.com] has ended non-interactive session [9febab45-6491-11e9-80a1-427cfde50f5a] on node [undefined] 
+            User [admin@example.com] has ended an non-interactive session [9febab45-6491-11e9-80a1-427cfde50f5a] on node [undefined]
```
Comment (webvictim) [inline packages/teleport/src/cluster/components/Audit/AuditEvents/__snapshots__/AuditEvents.story.test.tsx.snap:1531]: Not sure if this should be `a non-interactive` too?

File: packages/teleport/src/cluster/components/Audit/AuditEvents/__snapshots__/AuditEvents.story.test.tsx.snap [hunk 3]
```diff
@@ -1361,7 +1493,7 @@ exports[`rendering of Events 1`] = `
           <td
             style="word-break: break-word;"
           >
-            User [admin@example.com] has ended non-interactive session [9febab45-6491-11e9-80a1-427cfde50f5a] on node [undefined] 
+            User [admin@example.com] has ended an non-interactive session [9febab45-6491-11e9-80a1-427cfde50f5a] on node [node-hostname]
```
Comment (webvictim) [inline packages/teleport/src/cluster/components/Audit/AuditEvents/__snapshots__/AuditEvents.story.test.tsx.snap:1496]: Not sure if this should be `a non-interactive` too?

File: packages/teleport/src/cluster/components/Audit/AuditEvents/__snapshots__/AuditEvents.story.test.tsx.snap (hunk 2 above)
Comment (kimlisa) [reply packages/teleport/src/cluster/components/Audit/AuditEvents/__snapshots__/AuditEvents.story.test.tsx.snap]: thank you, i forgot to run my tests before commit :)

File: packages/teleport/src/cluster/components/Audit/AuditEvents/__snapshots__/AuditEvents.story.test.tsx.snap (hunk 3 above)
Comment (kimlisa) [reply packages/teleport/src/cluster/components/Audit/AuditEvents/__snapshots__/AuditEvents.story.test.tsx.snap]: updated

**Pull request merged**